datatypes-jws/src/commonMain/kotlin/at/asitplus/crypto/datatypes/jws/JsonWebKey.kt

```diff
@@ -158,18 +164,22 @@ fun CryptoPublicKey.toJsonWebKey(): JsonWebKey =
                 curve = curve,
                 x = x,
                 y = y
-            ).apply { jwkId = identifier }
+            )
 
         is CryptoPublicKey.Rsa ->
             JsonWebKey(
                 type = JwkType.RSA,
                 keyId = jwkId,
                 n = n,
                 e = e.encodeToByteArray()
-            ).apply { jwkId = identifier }
+            )
```

Comment:
it moved to where it should belong https://github.com/a-sit-plus/kmp-crypto/pull/19/files#diff-7d925f2ef6d4cc43e67075adfb1690d89a3cfe330a6174afbc7f4bcb0b303843R151

(but i seem to have messed up anyways, since this was already merged into main after approval from @n0900)